src/views/partials/signout-bar.njk

```diff
@@ -4,5 +4,4 @@
         <span class="content-email" id="signed-in-user">{{email}}</span>
         <a class="govuk-link" href="{{ExternalUrls.SIGNOUT}}" data-event-id="sign-out-button-selected" id="user-signout">Sign Out</a>
     </div>
-<script src="{{ cdnHost }}/javascripts/app/session-timeout.js"></script>
```

Comment:
Don't we need this though?

Reply:
[I moved it into the footer](https://github.com/companieshouse/presenter-account-web/blob/797b53f323ec8b4dab9a0026997641d1ace6524c/src/views/partials/__footer.njk#L48) with the rest of the scripts so they only run after the HTML is loaded. If preferred, I could keep it here with the signout bar and add the `defer` attribute to achieve the same effect, but keep it with the signout bar HTML.